Development-Guide/CAN-Filter-Guide/README.adoc

```diff
@@ -0,0 +1,61 @@
+= CAN Message Filter Guide
+
+== Introduction
+This is a basic guide on how to make a CAN filter.
+There are two components to a CAN filter: an ID and a Mask.
+The ID is the unique identifier for the address of the message(s) being looked for, and the Mask determines which bits of the ID are important to sift through. 
+
+IMPORTANT: A Mask must be able to adapt to future additions to a CAN network. 
+It must be made *as tight as possible for longevity*.
+
+There is no set order of steps to follow, but to get started, determine the ID, then determine the Mask.
+A very useful tip is to convert the hex format into binary format, determine the ID and Mask, and then convert those back into hex.
+
+
+== Determine the ID
+The ID will be the unique identifier for the address of the message(s) being received.
+
+. The ID should be very specific and contain the important bits that will be compared to the incoming message.
+For example, if one is looking for a message from a system that sends out messages with address 0b0000_1111(0x0F) then the ID will be 0b0000_1111(0x0F) because there is only one type of address that is wanted.
```

Comment:
Let's have the the addresses emphasized by either boldness, itialics, or backticks to improve readability.